packages/blocks/src/api/raw-handling/list-reducer.js

```diff
@@ -8,7 +8,9 @@ function isList( node ) {
 }
 
 function shallowTextContent( element ) {
-	return [ ...element.childNodes ]
+	// On mobile DOM implmentation NodeList is not spreadable so we need to convert it to an array.
+	const nodesArray = Array.from( element.childNodes );
```

Comment:
Why introduce the constant?

Reply:
Ok I removed it and using directly the Array.from result